config/find-eu-exit-guidance-business.yml

```diff
@@ -12,7 +12,7 @@ rendering_app: finder-frontend
 details:
   beta: false
   document_noun: publication
-  subscription_list_title_prefix: Find EU Exit guidance for your business
+  subscription_list_title_prefix: 'Find EU Exit guidance for your business '
```

Comment:
This makes me sad

Reply:
Unfortunately precedent has already been set here as the current finder signups (albeit only single facet ones) include a trailing space on this value.